src/actions/purchaseDomains.ts

```diff
@@ -76,27 +76,13 @@ export const purchaseDomains = async (
   // If using a Gnosis safe as the seller wallet you must
   // provide the implementation address for the tx accessList
   const sellerWallet = await contract.sellerWallet();
-  const sellerContract = new ethers.Contract(sellerWallet, abi, contract.provider);
+  const sellerContract = new ethers.Contract(
```

Comment:
I should've caught this in the previous PR, but this is prone to failure. If `sellerWallet` is not a contract then this will throw an error and the user will unable to purchase a domain.